docs/source/bricks.rst

```diff
@@ -90,7 +90,11 @@ Examples:
 The ``partition_email`` function partitions ``.eml`` documents and works with exports
 from email clients such as Microsoft Outlook and Gmail. The ``partition_email`` 
 takes a filename, file-like object, or raw text as input and produces a list of
-document ``Element`` objects as output.
+document ``Element`` objects as output. Also ``content_source`` can be set to ``text/html``
+(default) or ``text/plain`` to process the html or plain text version of the email, respectively.
+In order for ``partition_email`` to also return the header information (e.g. sender, recipient,
+attachment, etc.), ``get_meta_data`` must be set to ``True``. Returns tuple with body elements
```

Comment:
Minor nit: `get_meta_data` -> `get_metadata` for consistency with `unstructured/file_utils/metadata.py`

Reply:
Actually per the other comment below, I think we can do `include_headers` and preprend the header outputs since they're `Element` objects. 

We can add `.eml` logic `unstructured/file_utils/metadata.py` in a separate PR later.